mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -1464,8 +1468,15 @@ public void copiedMessageShouldAllBeIndexed() throws Exception {
         SearchQuery searchQuery = new SearchQuery();
 
         StoreMessageManager newBox = (StoreMessageManager) storeMailboxManager.getMailbox(newBoxId, session);
-        assertThat(messageSearchIndex.search(session, newBox.getMailboxEntity(), searchQuery))
-            .hasSize(9);
+
+        Awaitility.with()
+            .pollInterval(Duration.ONE_HUNDRED_MILLISECONDS)
+            .and().with()
+            .pollDelay(new Duration(1, TimeUnit.MILLISECONDS))
```

Comment:
why do you require a delay at all? 1 ms is so small that it should not make a difference, is it?

Reply:
to ensure the test is quickly run, else it waits for the poll interval (I just took the same configuration than for JMAP integration tests)